src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { textKilo } from '../../styles/style-helpers';
+
+const baseStyles = css`
+  label: tooltip;
+  position: relative;
+  display: inline-block;
+
+  &:hover {
+    > span {
+      visibility: visible;
+      opacity: 1;
+    }
+  }
+`;
+
+const arrowAligns = theme => ({
+  center: `left: 50%`,
+  right: `left: ${theme.spacings.mega}`,
+  left: `right: ${theme.spacings.mega}`
+});
+
+const tooltipAlign = {
+  center: `
+    left: 0;
+    right: 0;
+    margin-left: auto;
+    margin-right: auto;
+  `,
+  right: 'left: 50%;',
+  left: 'right: 50%;'
+};
+
+const tooltipTextStyles = ({ align, theme }) => css`
+  label: tooltip_text;
+  visibility: hidden;
+  opacity: 0;
+  width: max-content;
+  min-width: 60px;
+  background-color: ${theme.colors.black};
+  color: ${theme.colors.white};
+  text-align: center;
+  border-radius: ${theme.borderRadius.mega};
+  padding: ${theme.spacings.byte};
+  position: absolute;
+  z-index: 1;
+  bottom: 125%;
+  ${tooltipAlign[align]};
+  transition: opacity 0.3s;
+  ${textKilo({ theme })};
+
+  &:after {
```

Comment:
Please use `::` for `before` and `after`.

Reply:
Sure thing :)